### PR TITLE
Add business finder integration

### PR DESCRIPTION
**Pull request by dpdrabla**

### CODE REVIEW

**Review by abhifetch — APPROVED** [review]

File: integrations/business finder/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# uAgent Business Finder Integrations Examples 🤑
+
+This repository contains examples of business finder integrations using three agents: `business_finder_agent`, `business_details_agent` and `user_agent`.
+
+1. `user_agent` : This agent takes request as details from user (city and category), type of business they want to look for.
```
Comment (FelixNicolaeBucsa) [inline integrations/business finder/README.md:5]: `user_agent` : This --> `user_agent`: This

File: integrations/business finder/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# uAgent Business Finder Integrations Examples 🤑
+
+This repository contains examples of business finder integrations using three agents: `business_finder_agent`, `business_details_agent` and `user_agent`.
+
+1. `user_agent` : This agent takes request as details from user (city and category), type of business they want to look for. 
+
+2. `business_finder_agent`: This agent takes city and category from user, fetch city's location coordinates using [GeoCoding API](https://rapidapi.com/apininjas/api/geocoding-by-api-ninjas/) and passing it to [Local Business Finder](https://rapidapi.com/alreadycoded/api/local-business-listing-finder/) to fetch list of ten businesses names.
```
Comment (FelixNicolaeBucsa) [inline integrations/business finder/README.md:7]: fetch --> fetches

passing --> passes

File: integrations/business finder/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+3. `business_details_agent`: This agent gets list of business names from business finder agent and takes input from user, which business they want details about. It retrieves the details of business using API's used in `business_finder_agent` and sends it to `business_finder_agent` which in turn sends it to user.
```
Comment (FelixNicolaeBucsa) [inline integrations/business finder/README.md:9]: takes input from user, which business --> takes input from user about which business

Comment (FelixNicolaeBucsa) [reply integrations/business finder/README.md:9]: details of business --> details of the business

File: integrations/business finder/src/business_detail_agent.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# importing required libraires
+import requests
+from ai_engine import UAgentResponse, UAgentResponseType
+from uagents import Agent, Model, Context, Protocol
+from uagents.setup import fund_agent_if_low
+from models import Response, Response_list, Location
+
+#defining business details angent
```
Comment (FelixNicolaeBucsa) [inline integrations/business finder/src/business_detail_agent.py:8]: #defining business details angent --> # Defining business details angent

Comment (FelixNicolaeBucsa) [reply integrations/business finder/src/business_detail_agent.py:8]: Also for below comments within code snippet

**Review by FelixNicolaeBucsa — COMMENTED** [review]
Left some minor comments

**Comment by dpdrabla** [comment]
@devjsc can you please review it.

**Review by FelixNicolaeBucsa — APPROVED** [review]

File: integrations/.DS_Store
Comment (devjsc) [inline integrations/.DS_Store:1]: Please remove, update .gitignore

File: integrations/business finder/src/business_finder_agent.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    url = "https://local-business-data.p.rapidapi.com/search-nearby"
+    querystring = {"query":category,"lat":latitude,"lng":longitude,"limit":"10","language":"en"}
+    headers = {
+	    "X-RapidAPI-Key": "BUSINESS_API_KEY",
+	    "X-RapidAPI-Host": "local-business-data.p.rapidapi.com"
+    }
+    response = requests.get(url, headers=headers, params=querystring)
+    business = []
+    data = response.json()['data']
+    print(1)
```
[the remark this answers is not in the document]
Comment (dpdrabla) [reply integrations/business finder/src/business_finder_agent.py]: removed

File: integrations/.DS_Store
Comment (dpdrabla) [reply integrations/.DS_Store:1]: added .DS_Store to .gitignore

**Pull request merged**